src/main/java/org/variantsync/diffdetective/variation/NodeType.java

```diff
@@ -1,7 +1,8 @@
 package org.variantsync.diffdetective.variation;
 
-import org.variantsync.diffdetective.variation.diff.DiffNode; // For Javadoc
-import org.variantsync.diffdetective.variation.tree.VariationNode; // For Javadoc
+import org.variantsync.diffdetective.feature.AnnotationType;
+import org.variantsync.diffdetective.variation.diff.DiffNode;
+import org.variantsync.diffdetective.variation.tree.VariationNode;
```

Comment:
Please keep the comments `// For Javadoc`.

Reply:
They are automatically removed by Intellij... I disabled import optimization.